### PR TITLE
Create 9.upgrade-nebula-cluster.md

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

**Review by foesa-yang — APPROVED** [review]

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    imagePullPolicy: Always
+  ```
+
+3. Run the following command to apply the version update to the cluster CR.
+   
+  ```bash
+  kubectl apply -f apps_v1alpha1_nebulacluster.yaml
+  ```
+
+4. After waiting for about 2 minutes, run the following command to see the image versions of the services in the cluster have been changed to {{nebula.branch}}.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:124]:
```suggestion
4. After waiting for about 2 minutes, run the following command to see if the image versions of the services in the cluster have been changed to {{nebula.branch}}.
```

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+## Upgrade a Nebula Graph cluster with Helm
+
+### Prerequisites
+
+You have created a Nebula Graph cluster with Helm. For details, see [Create a Nebula Graph cluster with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md).
+
+### Steps
+
+1. Update information of available charts locally from chart repositories.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:146]:
```suggestion
1. Update the information of available charts locally from chart repositories.
```

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  ```bash
+  helm upgrade "${NEBULA_CLUSTER_NAME}" nebula-operator/nebula-cluster \
+      --namespace="${NEBULA_CLUSTER_NAMESPACE}" \
+      --set nameOverride=${NEBULA_CLUSTER_NAME} \
+      --set nebula.version={{nebula.branch}}
+  ```
+
+  The value of `--set nebula.version` specifies the version of the cluster you want to upgrade to.
+
+4. After waiting for a while, run the following command to check the status and version of the upgraded cluster.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:172]:
```suggestion
4. Run the following command to check the status and version of the upgraded cluster.
```

`After waiting for a while` is of no help. What do you think?
Either we provide the output of a successful upgrade, or we let the users judge by themselves whether the upgrade is successful or not.

Comment (abby-cyber) [reply docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md]: exactly agree with that

**Review by randomJoe211 — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**